extension_point_customercredit_checkout/react/components/CheckoutListener.js

```diff
@@ -167,19 +164,25 @@ class CheckoutListener extends Component {
         const creditControl = data.creditControl
         const creditAccount = creditControl.creditAccounts.value.accounts[0]
         const deadlines = creditControl.deadlines.value.deadlines
-        const minimumInstallmentValue =
-          (creditControl.minimumInstallmentValue &&
-            creditControl.minimumInstallmentValue.value) ||
-          0
 
         if (!creditControl || !deadlines || !creditAccount) {
           this.setState({ dataVerified: true })
           return this.manageError({
             response: { data: { code: 404 } },
-            step: 'fetchDeadlines',
+            step: 'verifyAccount',
           })
         }
 
+        if (!window.vtexjs.checkout.orderForm.loggedIn) {
```

Comment:
Não é bom verificar um parametro de cada vez pra nao estourar um `.something of undefined`?
```suggestion
        if (window && window.vtexjs && window.vtexjs.checkout && window.vtexjs.checkout.orderForm && !window.vtexjs.checkout.orderForm.loggedIn) {
```

Reply:
Boa. é verificado antes se o orderform existe ja. vi agora 😉